src/main/java/com/ays/assignment/model/dto/response/AssignmentSavedResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.ays.assignment.model.dto.response;
+
+import com.ays.common.model.AysPhoneNumber;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class AssignmentSavedResponse {
+
+    private String id;
+    private String description;
+    private String firstName;
+    private String lastName;
+    private AysPhoneNumber phoneNumber;
+    private double latitude;
+    private double longitude;
+}
```

Comment:
Bu akışta herhangi bir cevap dönmemize ihtiyaç olmadığı için buradaki objeyi silebiliriz

Reply:
`AssignmentSavedResponse` objeyi sildim.